### PR TITLE
[CEM-19380]: Added configuration for zookeeper admin server

### PR DESCRIPTION
**Pull request by lrebeja**

### CODE REVIEW

File: pkg/apis/contrail/v1alpha1/templates/zookeeper_config.go [hunk 1]
```diff
@@ -4,24 +4,26 @@ import (
...
-dataDir=/var/lib/zookeeper
+var ZookeeperStaticConfig = template.Must(template.New("").Parse(`dataDir=/var/lib/zookeeper
 tickTime=2000
 initLimit=5
 syncLimit=2
 maxClientCnxns=60
 maxSessionTimeout=120000
-admin.enableServer=false
+admin.enableServer={{ .AdminEnableServer }}
+admin.serverPort={{ .AdminSeverPort }}
```
Comment (psykulsk) [inline pkg/apis/contrail/v1alpha1/templates/zookeeper_config.go:20]: Please fix AdminSeverPort -> AdminServerPort

File: pkg/apis/contrail/v1alpha1/zookeeper_types.go [hunk 2]
```diff
@@ -92,7 +95,19 @@ func (c *Zookeeper) InstanceConfiguration(request reconcile.Request, confCMName
 	}
 	confCMData["log4j.properties"] = configtemplates.ZookeeperLogConfig
 	confCMData["configuration.xsl"] = configtemplates.ZookeeperXslConfig
-	confCMData["zoo.cfg"] = configtemplates.ZookeeperStaticConfig
+	var zookeeperConfigBuffer bytes.Buffer
+	err = configtemplates.ZookeeperStaticConfig.Execute(&zookeeperConfigBuffer, struct {
+		AdminEnableServer string
+		AdminSeverPort    string
```
Comment (psykulsk) [inline pkg/apis/contrail/v1alpha1/zookeeper_types.go:101]: Please fix AdminSeverPort -> AdminServerPort

**Review by psykulsk — CHANGES_REQUESTED** [review]
Can you add a unit test for this ZookeeperStaticConfig that will check the default values and also a case with customized values?

**Review by vijayrap — APPROVED** [review]
LGTM.

**Comment by psykulsk** [comment]
recheck ha

**Review by psykulsk — APPROVED** [review]

**Pull request merged**